component/attr_cache/attr_cache_test.go

```diff
@@ -898,15 +898,22 @@ func (suite *attrCacheTestSuite) TestCreateFile() {
 
 	_, err = suite.attrCache.CreateFile(options)
 	suite.assert.Nil(err)
-	assertNotInCache(suite.assert, suite.attrCache, path)
+	assertExists(suite, options.Name)
```

Comment:
Why did we need to change this? Was createFile not creating an entry in the cache before?

Reply:
Correct. Previously, `CreateFile` would invalidate the attributes for the file, since that's what they would do any time the file changed in cloud storage.